vocabularies/quality.ttl

```diff
@@ -0,0 +1,685 @@
+  rdfs:label "Efficiency"@en ;
+  skos:definition "The degree to which data has attributes that can be processed and provide the expected levels of performance by using the appropriate amounts and types of resources in a specific context of use."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Efficiency"@en ;
+  dqv:inCategory dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:broader dq-iso-25012:inherentSystemDependentDataQuality ;
+.
+dq-iso-25012:inherentDataQuality
+  a dqv:Category ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Inherent data quality"@en ;
+  skos:prefLabel "Inherent data quality"@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+.
+dq-iso-25012:inherentSystemDependentDataQuality
+  a dqv:Category ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Inherent system-dependent data quality"@en ;
+  skos:related dq-iso-25012:inherentDataQuality ;
+  skos:related dq-iso-25012:systemDependentDataQuality ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Inherent and System-Dependent Data Quality"@en ;
+.
+dq-iso-25012:portability
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Portability"@en ;
+  skos:definition "The degree to which data has attributes that enable it to be installed, replaced or moved from one system to another preserving the existing quality in a specific context of use."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Portability"@en ;
+  dqv:inCategory dq-iso-25012:systemDependentDataQuality ;
+  skos:broader dq-iso-25012:systemDependentDataQuality ;
+.
+dq-iso-25012:precision
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Precision"@en ;
+  skos:definition "The degree to which data has attributes that are exact or that provide discrimination in a specific context of use."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Precision"@en ;
+  dqv:inCategory dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:broader dq-iso-25012:inherentSystemDependentDataQuality ;
+.
+dq-iso-25012:recoverability
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Recoverability"@en ;
+  skos:definition "The degree to which data has attributes that enable it to maintain and preserve a specified level of operations and quality, even in the event of failure, in a specific context of use."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Recoverability"@en ;
+  dqv:inCategory dq-iso-25012:systemDependentDataQuality ;
+  skos:broader dq-iso-25012:systemDependentDataQuality ;
+.
+dq-iso-25012:systemDependentDataQuality

```

Comment:
No definition or documentation of meaning or examples.

Reply:
updated from https://iso25000.com/index.php/en/iso-25000-standards/iso-25012